core/trino-main/src/main/java/io/trino/sql/routine/SqlRoutineCompiler.java

```diff
@@ -334,8 +335,8 @@ public BytecodeNode visitBlock(IrBlock node, Scope scope)
             LabelNode breakLabel = new LabelNode("break");
 
             if (node.label().isPresent()) {
```

Comment:
Could switch this to `.ifPresent(label -> {` to avoid repeated `node.label().get()` calls.

Reply:
i understand this is for performance reasons?
ifPresent requires a lambda (with capture), which is more expensive than direct if. i am not sure if i should replace in both places or only this one?

Reply:
fixed here